src/applications/personalization/rated-disabilities/sass/rated-disabilities.scss

```diff
@@ -10,13 +10,37 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 .rated-disability-item-container {
     background-color: #f1f1f1;
```

Comment:
We have a background color class for this `vads-u-background-color--gray-lightest` [here](https://design.va.gov/utilities/background-color)